src/app/armory/Armory.m.scss

```diff
@@ -27,6 +27,15 @@
   }
 }
 
+.exportButton {
+  margin-bottom: 8px;
+
+  &.exportButton > button {
+    font-size: 14px;
+    padding: 6px 12px;
+  }
+}
```

Comment:
This CSS makes no sense. It is trying to match `.exportButton.exportButton > button`.

Also there's no need to set the font size here.